CefSharp.MinimalExample.WinForms/BrowserForm.cs

```diff
@@ -156,6 +156,11 @@ private void HandleToolStripLayout()
 
         private void ExitMenuItemClick(object sender, EventArgs e)
         {
+            // Prevent a race condition where browser.Dispose tears down it's control
+            // (before all browser cleanup has finished)
+            // https://github.com/cefsharp/CefSharp/issues/1574
+            // There are better/more complicated ways to handle this too
```

Comment:
Should we be demonstrating one of those instead?

It's should now be a simple one liner, either of the following should work:

```c#
browser.LifeSpanHandler = new CefSharp.WinForms.Handler.LifeSpanHandler();

browser.LifeSpanHandler = CefSharp.WinForms.Handler.LifeSpanHandler
    .Create()
    .Build();
```